frontend/apps/app/src/trigger/jobs.ts

```diff
@@ -1,23 +1,43 @@
 import { processGenerateReview } from '@/src/functions/processGenerateReview'
+import {
+  type SavePullRequestPayload,
+  processSavePullRequest,
+} from '@/src/functions/processSavePullRequest'
 import { logger, task } from '@trigger.dev/sdk/v3'
 import type { GenerateReviewPayload, ReviewResponse } from '../types'
 
 export const savePullRequestTask = task({
   id: 'save-pull-request',
   run: async (payload: {
     pullRequestNumber: number
-    projectId: number
+    projectId: number | undefined
+    owner: string
+    name: string
     repositoryId: number
-    schemaChanges: Array<{
-      filename: string
-      status: 'added' | 'modified' | 'deleted'
-      changes: number
-      patch: string
-    }>
   }) => {
     logger.log('Executing PR save task:', { payload })
-    await generateReviewTask.trigger(payload)
-    return { success: true }
+
+    try {
+      const result = await processSavePullRequest({
+        prNumber: payload.pullRequestNumber,
+        owner: payload.owner,
+        name: payload.name,
+        repositoryId: payload.repositoryId,
+      } as SavePullRequestPayload)
```

Comment:
Please do not use `as` if possible. Was it necessary?

Reply:
Unnecessary.  Applying https://github.com/liam-hq/liam/pull/869#issuecomment-2720352670 will solve that. 🙏

Reply:
Looks good, Thank you 👍🏻